app/models/activity.rb

```diff
@@ -277,6 +277,24 @@ def own_and_descendants_transactions
     Transaction.where(parent_activity_id: activity_ids)
   end
 
+  def reportable_transactions_for_level
+    reportable_transactions = if programme?
+      own_and_descendants_transactions
+    else
+      transactions
+    end.order("date DESC")
+
+    return reportable_transactions unless programme?
+
+    spend_by_financial_quarter(reportable_transactions)
```

Comment:
This tripped me up a bit - would it be clearer to return the `spend_by_financial_quarter` on the `own_and_descendants_transactions` if the activity is a programme, otherwise return the transactions? I may be missing something here though, so let me know if I am 👍 

```suggestion
    if programme?
      spend_by_financial_quarter(own_and_descendants_transactions.order("date DESC"))
    else
      transactions.order("date DESC")
    end
```

Reply:
Also, what happens if the activity is a fund?

Reply:
Nice suggestion, that's much clearer.

I don't think it's possible to report on a whole fund.